certbot/_internal/plugins/__init__.py

```diff
@@ -0,0 +1 @@
+"""Certbot client.plugins."""
```

Comment:
I think `client.plugins` here refers to when the plugins directory was contained in a `client` directory (see 2d848994f4aec4a640ac6c526d9ea28f68dc8ff2 if you're curious).

I don't think we should we should copy this over to this new file. Bonus points for fixing it in `certbot/plugins/__init__.py` too, but I'm also fine doing that later since it's not technically related to this PR.